util/libcrypto.num

```diff
@@ -4446,3 +4446,53 @@ RSA_set0_multi_prime_params             4390	1_1_1	EXIST::FUNCTION:RSA
+ADMISSION_SYNTAX_free                   4402	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_free                    4403	1_1_1	EXIST::FUNCTION:
+i2d_PROFESSION_INFO                     4404	1_1_1	EXIST::FUNCTION:
+i2d_ADMISSION_SYNTAX                    4405	1_1_1	EXIST::FUNCTION:
+ADMISSIONS_get0_admissionAuthority      4406	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_get0_addProfessionInfo  4407	1_1_1	EXIST::FUNCTION:
+ADMISSION_SYNTAX_set0_contentsOfAdmissions 4408	1_1_1	EXIST::FUNCTION:
+ADMISSIONS_set0_professionInfos         4409	1_1_1	EXIST::FUNCTION:
+ADMISSIONS_set0_namingAuthority         4410	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_set0_authorityText     4411	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_set0_addProfessionInfo  4412	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_get0_namingAuthority    4413	1_1_1	EXIST::FUNCTION:
+d2i_PROFESSION_INFO                     4414	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_get0_authorityId       4415	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_it                      4416	1_1_1	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:
+PROFESSION_INFO_it                      4416	1_1_1	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:
+PROFESSION_INFO_set0_professionItems    4417	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_it                     4418	1_1_1	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:
+NAMING_AUTHORITY_it                     4418	1_1_1	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:
+ADMISSION_SYNTAX_new                    4419	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_set0_authorityId       4420	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_get0_professionOIDs     4421	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_free                   4422	1_1_1	EXIST::FUNCTION:
+ADMISSIONS_set0_admissionAuthority      4423	1_1_1	EXIST::FUNCTION:
+d2i_ADMISSION_SYNTAX                    4424	1_1_1	EXIST::FUNCTION:
+ADMISSIONS_it                           4425	1_1_1	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:
+ADMISSIONS_it                           4425	1_1_1	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:
+ADMISSIONS_get0_namingAuthority         4426	1_1_1	EXIST::FUNCTION:
+ADMISSION_SYNTAX_get0_contentsOfAdmissions 4427	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_set0_professionOIDs     4428	1_1_1	EXIST::FUNCTION:
+i2d_NAMING_AUTHORITY                    4429	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_get0_professionItems    4430	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_set0_authorityURL      4431	1_1_1	EXIST::FUNCTION:
+ADMISSIONS_get0_professionInfos         4432	1_1_1	EXIST::FUNCTION:
+i2d_ADMISSIONS                          4433	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_get0_registrationNumber 4434	1_1_1	EXIST::FUNCTION:
+ADMISSION_SYNTAX_get0_admissionAuthority 4435	1_1_1	EXIST::FUNCTION:
+d2i_NAMING_AUTHORITY                    4436	1_1_1	EXIST::FUNCTION:
+d2i_ADMISSIONS                          4437	1_1_1	EXIST::FUNCTION:
+PROFESSION_INFO_set0_registrationNumber 4438	1_1_1	EXIST::FUNCTION:
+NAMING_AUTHORITY_get0_authorityURL      4439	1_1_1	EXIST::FUNCTION:
```

Comment:
github ate my other other comment which was: Is this for backport to 1.1.0? If so the version numbers here would need updating.